src/connectors/meraki.py

```diff
@@ -0,0 +1,194 @@
+        ingest_type = 'client'
+    elif (table_name.endswith('DEVICE') or table_name.endswith('device')):
+        ingest_type = 'device'
+    
+    landing_table = f'data.{table_name}'
+
+    timestamp = datetime.utcnow()
+    api_key = options['api_key']
+    whitelist = options['network_id_whitelist']
+
+    for network in whitelist:
+        
+        try:
+            devices = get_data(f"https://api.meraki.com/api/v0/networks/{network}/devices", api_key)
+        except requests.exceptions.HTTPError as e:
+            log.error(f"{network} not accessible, ")
+            log.error(e)
+            continue
+
+        if (ingest_type == 'device'):
+            db.insert(
+                landing_table,
+                values=[(
+                    timestamp,
+                    device,
+                    device.get('serial',None),
+                    device.get('address',None),
+                    device.get('name',None),
+                    device.get('networkId',None),
+                    device.get('model',None),
+                    device.get('mac',None),
+                    device.get('lanIp',None),
+                    device.get('wan1Ip',None),
+                    device.get('wan2Ip',None),
+                    device.get('tags',None),
+                    device.get('lng',None),
+                    device.get('lat',None),
+                ) for device in devices],
+                select=db.derive_insert_select(LANDING_TABLE_COLUMNS_DEVICE),
+                columns=db.derive_insert_columns(LANDING_TABLE_COLUMNS_DEVICE)
+            )
+            log.info(f'Inserted {len(devices)} rows ({landing_table}).')
+            yield len(devices)
+
+        elif (ingest_type == 'client'):
+            
+            for device in devices:
+                serial_number = device['serial']
+                
+                try:
+                    clients = get_data(f"https://api.meraki.com/api/v0/devices/{serial_number}/clients", api_key)
+                except requests.exceptions.HTTPError as e:
+                    log.error(f"{network} not accessible, ")
+                    log.error(e)
+                    continue
+                
+                db.insert(
+                    landing_table,
+                    values=[(
+                        timestamp,
+                        client,
+                        client.get('id',None),
+                        client.get('mac',None),
+                        client.get('description',None),
+                        client.get('mdnsName',None),
+                        client.get('dhcpHostname',None),
+                        client.get('ip',None),
+                        client.get('switchport',None),
+                        None if (client.get('vlan',None) == '') else client.get('vlan', None),
+                        None if (client.get('usage',None).get('sent',None) == '') else client.get('usage',None).get('sent',None),
```

Comment:
this seems like it might fail if client.get('usage') returns None. 
```suggestion
                        None if (client.get('usage',{}).get('sent',None) == '') else client.get('usage',{}).get('sent',None),
```